src/adm/tiles.rs

```diff
@@ -87,20 +87,27 @@ pub struct Tile {
     pub url: String,
     pub click_url: String,
     pub image_url: String,
+    pub image_size: Option<u32>,
+    pub image_metrics: ImageMetrics,
     pub impression_url: String,
     pub position: Option<u8>,
+    #[serde(skip_serializing)]
+    pub new: bool,
 }
 
 impl Tile {
-    pub fn from_adm_tile(tile: AdmTile, position: Option<u8>) -> Self {
+    pub fn from_adm_tile(tile: AdmTile, position: Option<u8>, new: bool) -> Self {
         Self {
             id: tile.id,
             name: tile.name,
             url: tile.advertiser_url,
             click_url: tile.click_url,
             image_url: tile.image_url,
+            image_size: None,
```

Comment:
Is this field still needed when `ImageMetrics` has a size?

We _could_ also cleanup this response output but it's not terribly important (nor do I know how it would affect the client impl cc: @ncloudioj)

e.g. 
```json
  "click_url": "...",
  image: {
    "url": "...",
    "size": "...",
    "width": "etc",
  }
```

Reply:
Part of the confusion is that "size" apparently is different. The UA wants "size" to be the width/height. We want "size" to be the length of the image (to construct the file name).

Reply:
Firefox only expects the `image_url` and the `image_width` from Contile. By historical reason, we reference `image_width` as `image_size` in the Fx codebase.

Alternatively, We can also leave out `image_width` from the response payload if Contile can enforce this invariant:
Contile would only provide the `image_url` field in the response if `image.width == image.height &&  96 <= image.width <= 256`.

So when we see `image_url` in the response, we know that the image is of the valid format, we can simply assign the minimal favicon width to it, and Firefox knows how to render it properly.

@jrconlin Let me know which one you prefer so that I can adjust my client patch accordingly.

Reply:
I'll add this as a comment to the code and only return `image_url` and `image_size`.

Reply:
sounds good!

Reply:
> Alternatively, We can also leave out `image_width` from the response payload if Contile can enforce this invariant:
> Contile would only provide the `image_url` field in the response if `image.width == image.height && 96 <= image.width <= 256`.

@jrconlin It looks like we're doing exactly this, so it sounds like we could only return `image_url`?

Reply:
I'd like to return both, since the size could vary between 96 and 128 px.
(I'll also say that previous experience has taught me to return more info than I need in these cases, but I'll defer to the folk doing the work.)

Reply:
Agreed, let's also return `image_size` other than letting UA hardcode it with `MIN_FAVICON_SIZE (96)`.

BTW, Dao and Amy have reviewed the client-side [patch](https://phabricator.services.mozilla.com/D119923), I am landing it to nightly now.